pkg/compact/compact.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/pkg/errors"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/prometheus/pkg/labels"
-	promlables "github.com/prometheus/prometheus/pkg/labels"
+	promlabels "github.com/prometheus/prometheus/pkg/labels"
```

Comment:
Ups, I missed this while upgrading to Prometheus with unified labels packages. This line can be removed and we can use `labels` below (: Thanks for spotting!